.github/workflows/code-quality-checks.yml

```diff
@@ -1,7 +1,7 @@
 name: Code Quality Checks
 on: [push]
 jobs:
-  run-tests:
+  run-unit-tests:
```

Comment:
what does unit test mean in this context? 

Is this mock unit test without the need for databricks account?
or is it integration test e2e which require databricks account

Reply:
`unit` means no databricks account is required.
`e2e` means databricks account is required.

Reply:
I'm updating the CONTRIBUTING doc with this info.

Reply:
thanks Jesse.